### ISSUE TITLE
feat: Branding page, with brand assets

### ISSUE THREAD

**Issue by MildTomato** [supabase/supabase#1110]
# Feature request

## Is your feature request related to a problem? Please describe.

Add a page on the website that allows for anyone to download any of the Supabase brand asset.

website is located in `supabase/www/` in this repo.

As a first version, probably just displaying the artwork with a link to download `.png` and `.svg` would be sufficient.

examples of other asset pages
https://brand.mastercard.com/brandcenter/mastercard-brand-mark/downloads.html

we could also just compile all the assets into a zip folder and have the one link to download.
https://stripe.com/en-sg/newsroom/brand-assets

all the assets can be kept in /public/brand-assets/*
we can easily update the assets in here in case we need to update/amend them.

@MildTomato can provide asset artwork.

## Describe the solution you'd like

Page / route to be located at `https://supabase.io/brand-assets`

## Additional context

We would also like to provide badges, that would be useful for displaying on websites, and github repos.
although this is a nice to have for now - @joshenlim / @MildTomato can provide artwork for these.

**Comment by MildTomato** [comment]
also add a link in footer for this new page

**Comment by kevcodez** [comment]
Also, I couldn't find a light logo of the supabase brand logo (there is one on the docs page, but it has "Docs" attached), would be great to have an official light logo, too.

**Comment by awalias** [comment]
@hobbleabbas could be a good one for your list, someone on twitter was asking for the svg earlier I think. @MildTomato are these assets only on figma at the moment?

**Comment by MildTomato** [comment]
@awalias they are.. we should get at least the logos and icons into a folder in `www` though so a page can be made and nobody is waiting on me.

i'll throw it in a feature branch - `feature/brand-assets-page`
